src/consensus/raft/raft.h

```diff
@@ -132,6 +134,8 @@ namespace raft
       NodeId id,
       std::chrono::milliseconds request_timeout_,
       std::chrono::milliseconds election_timeout_,
+      std::shared_ptr<kv::AbstractTxEncryptor> encryptor_ =
```

Comment:
I wouldn't expect the `NullTxEncryptor` to be used here. Could this be `nullptr` instead?

Reply:
For testing purposes it is nicer to set a NullTxEncryptor by default instead of going to all of our unit tests and setting one manually. It also saves a lot of nullptr checks

Reply:
I think that using the `NullTxEncryptor` as default should be avoided, and checking for `nullptr` is fine (it's only required once in this file).